### PR TITLE
The first Gradle task (Step 2)

### PR DESCRIPTION
**Pull request by a1exanddrovich**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    testImplementation 'org.junit.jupiter:junit-jupiter-api:5.7.0'
+    testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.7.0'
+    implementation 'org.apache.commons:commons-lang3:3.10'
```
Comment (NatalliaKhadunai) [inline build.gradle]: We should put dependencies for the main source set first and then dependencies for the test source set.
So `implementation` dependencies should be located before `testImplementation` and `testRuntimeOnly` dependencies.

File: src/main/java/com/epam/task/first/StringUtils.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package com.epam.task.first;
+
+public class StringUtils {
+
+    private final static String ZERO = "0.0";
+    private final static String MINUS = "-";
+
+    public boolean isPositiveNumbers(String str) {
```
Comment (NatalliaKhadunai) [inline src/main/java/com/epam/task/first/StringUtils.java]: In the requirements the method has a different name `isPositiveNumber` (without `s`)

**Review by NatalliaKhadunai — COMMENTED** [review]
Left a couple of comments

File: build.gradle (hunk 1 above)
Comment (a1exanddrovich) [reply build.gradle]: So as I got first we write dependencies for the implementation and after that for the tests. Reordered it

File: src/main/java/com/epam/task/first/StringUtils.java (hunk 2 above)
Comment (a1exanddrovich) [reply src/main/java/com/epam/task/first/StringUtils.java]: Corrected, too

**Pull request merged**